RaduStanca/teme/tema-obiecte-3.3.7/script.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Tema-3-JavaScript-Obiecte
+ */
+
+/**
+ *ex.1- Scrieți o funcție care creează un obiect pentru a păstra informații despre rețeta ta preferată.
+
+Exemplu 1:
+
+Input: titlu = “Pizza”, portii = 3, ingrediente = [“sunca”, “cas”, “ketchup”]
+
+Output: { titlu: “Pizza”, portii: 3, ingrediente: [”sunca”, “cas”, “ketchup”] }
+ */
+
+
+const myFavoriteRecipe = (titlu, portii,ingrediente) => {
+    const myrecipe = {
+       titlu,
+       portii,
+       ingrediente,
+    };
+ 
+    console.log("titlu:", myrecipe.titlu);
+    console.log("portii:", myrecipe.portii);
+    console.log("ingrediente:", myrecipe.ingrediente);
+ 
+    return myrecipe;
+ };
+ 
+ myFavoriteRecipe("pizza", 3, ["sunca", "cas", "ketchup"])
+ 
+ // ex.2-Scrieți o funcție care returnează un obiect nou fără proprietățile furnizate
+ /**
+  * Input: obiect = { a: 1, b: 2 }, proprietate = “b”
+ 
+ Output: { a: 1 }
+  */
+ 
+ 
+    const returnsOnObject = (object, property) => {
```

Comment:
Numele functiei nu e foarte sugestiv. Daca il citesti in engleza, nu prea inseamna nimic. Un nume mai ok ar fi: deleteObjectProperty